### ISSUE TITLE
Cannot prebuild when the path contains spaces

### ISSUE THREAD

**Issue by antonino-u** [leavez/cocoapods-binary#88]
Thanks for your feedback!

If it's a bug:

- What's your environment:
  - run ` pod env | egrep 'CocoaPods |Xcode|cocoapods-binary' ` then paste the result here 

>    CocoaPods : 1.7.5
>        Xcode : 10.3 (10G8)
> cocoapods-binary      : 0.4.4
> plugin 'cocoapods-binary'
> 

- A demo project is helpful:
  - It will be very helpful if provides a demo project that reproduce the problem. A blank project with cocoapods-binary can be downloaded [here](https://github.com/leavez/cocoapods-binary/files/3086250/BinaryDemo.zip), which can be remodeled to a minimal problematic environment. Attach the demo.zip to the comments.

Add a space to your folder that contains your Podfile and, magically, binaries will no longer be prebuilt! (Delete your pods folder to force it to rebuild).
Unfortunately, the issue is that when prebuilding, the path that is eventually given to xcodebuild is not escaped.

**Comment by DimDL** [comment]
I can confirm I hit the same issue.

**Comment by MarcoBrescianini** [comment]
I can also confirm i hit the same issue

**Comment by MariuszWisniewski** [comment]
I can confirm same thing, managed to fix it on our side, but @antonino-u fix is better, so would be great if #89 could be merged.

**Comment by NathanielBlackburn** [comment]
Confirmed, still an issue, please consider merging #89.

**Comment by kajz3reczka** [comment]
yep, the problem still exists and #89 fix it. Maybe its time to merge :)
